collections/_posts/2022-10-17-DiffusionModel.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: review
+title: "Deep Unsupervised Learning using Nonequilibrium Thermodynamics"
+author: "Sophie Carneiro Esteves"
+cite:
+    authors: "Jascha Sohl-Dickstein, Eric A. Weiss, Niru Maheswaranathan and Surya Ganguli"
+    title:   "Deep Unsupervised Learning using Nonequilibrium Thermodynamics"
+    venue:   "International Conference on Machine Learning"
```

Comment:
It's a good idea to also specify the year of the publication (in the venue field). In this case:
```suggestion
    venue:   "International Conference on Machine Learning 2015"
```